client/packages/system/src/Item/Components/StockItemSearchInputWithStats/StockItemSearchInputWithStats.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, useEffect } from 'react';
+import {
+  useToggle,
+  useFormatNumber,
+  useTranslation,
+  styled,
+  Autocomplete,
+  defaultOptionMapper,
+} from '@openmsupply-client/common';
+import { useStockItemsWithStats } from '../../api';
+import { ItemRowWithStatsFragment } from '../../api/operations.generated';
+
+const ItemOption = styled('li')(({ theme }) => ({
+  color: theme.palette.gray.main,
+  backgroundColor: theme.palette.background.toolbar,
+}));
+
+const filterOptions = {
+  stringify: (item: ItemRowWithStatsFragment) => `${item.code} ${item.name}`,
+  // limit: 100, // unsure why we had the limit, and why so low. performance is ok for me
```

Comment:
hehe. perhaps time to remove my comment! 😉

Reply:
[b766752](https://github.com/openmsupply/open-msupply/pull/1134/commits/b7667522b27727fe7b2d202a5fb0e72e84faaaeb) ~